internal/vfs/vfstest/vfstest.go

```diff
@@ -168,6 +170,28 @@ func (m *mapFS) open(p canonicalPath) (fs.File, error) {
 	return m.m.Open(string(p))
 }
 
+func (m *mapFS) remove(path string) error {
+	canonical := m.getCanonicalPath(path)
+	canonicalString := string(canonical)
+	fileInfo, err := m.m.Stat(canonicalString)
+	if err != nil {
+		// file not found
```

Comment:
I'm not sure this is quite right; the Stat could fail due to a symlink being broken, for example.

I think that this code probably just needs to straight up _only_ use `m.m` and `m.symlinks`.